docs/index.md

```diff
@@ -0,0 +1,82 @@
+
+If you want to learn more about Scistream, please take a look at our [HPDC'22 paper](https://dl.acm.org/doi/abs/10.1145/3502181.3531475).
+
+## Documentation
+
+   - [Getting started](quickstart.md)
+   - [User Guide](user-guide/README.md)
+   - [Developer Guide](dev-guide/README.md)
+   - [Benchmarks](benchmarks/README.md)
+   - [About](about/README.md)
+
+## Project layout
+
+    poetry.lock         # Explicitly documents all Python dependencies using Poetry
+    pyproject.toml      # Python dependencie: Poetry equivalent to requirements.txt
+    mkdocs.yml          # Documentation configuration file.
+    docs/
+      index.md          # The documentation homepage.
+      ...               # Other markdown pages, images and other files.
+    deploy/
+      setup.sh          # Installation script
+    src/
+      proto/            # GRPC protocol specifications
+      appcontroller.py  # Application controller reference implementations
+      s2cs.py           # Scistream Control Server implementation
+      s2ds.py           # Scistream Dataplane server plugin
+      s2uc.py           # Scistream User Client implementation
+      utils.py          # Supporting code
+      ...               # other files
+    tests/
+      test_s2cs.py      # pytest tests
+      ...               # other tests
+
+### Service
+      The protocol should enable high-speed, memory-to-memory data streaming in scientific environments
+      by establishing streaming data channels between two remote facilities using our reference architecture:
+
+      ![alt text](figures/simple-arch.png "SciStream architecture")
+
+      Buffer-and-forward elements are run at the Science DMZ to create bridges between the Ethernet-based WAN and HPC interconnets where data producers/consumers may reside.
+
+### Software components
+      * **SciStream Data Server (S2DS):** software that runs on gateway nodes. It acts as a buffer-and-forward agent.
+      * **SciStream User Client (S2UC):** software that the end user and/or workflow engines/tools acting on behalf of the user interact with and provide relevant information (e.g., ID of a HPC job, ID of an experiment or data acquisition job on a scientific instrument, shared secret for secure communication with the user job (application) at the producer and consumer) to orchestrate end-to-end data streaming.
+      * **SciStream Control Server (S2CS):** a software running on one of the gateway nodes. It interacts with S2UC, data producer/consumer and S2DS.
+
+### Environment
+      * S2UC communicates with producer/consumer S2CS over a private LAN/WAN or the Internet
+      * S2CS and S2DS communicate over a LAN
+      * Messages can be lost or corrupted
+
+### Vocabulary of Messages
+      * **Requests:** { REQ, ReqListeners }
+      * **Responses:** { RESP, ProdLstn }
+      * **Commands:** { StartLstn, Hello, UpdateTargets, StartConn, Connect, REL }
+
+### Procedure Rules (Informal)
```

Comment:
This is probably outdated too

Reply:
Yeah I haven't deleted things yet. We can do that later.